superset-frontend/src/views/components/Menu.tsx

```diff
@@ -185,6 +185,11 @@ export function Menu({
   const screens = useBreakpoint();
   const uiConig = useUiConfig();
 
+  const getUploadMenulinks: MenuObjectProps[] =
+    menu &&
```

Comment:
do you need this check here for menu since you're using optional chaining below?

Reply:
No that was me trying to make typescript happy. I''ll remove that.